data_measurements/dataset_statistics.py

```diff
@@ -263,7 +263,31 @@ def __init__(
         # Try to pull from the hub to see if the cache already exists.
         try:
             if not isdir(self.cache_path) and self.dataset_cache_dir in [dataset_info.id.split("/")[-1] for dataset_info in list_datasets(author="datameasurements", use_auth_token=HF_TOKEN)]:
+                def get_size(start_path):
+                    total_size = 0
+                    for dirpath, dirnames, filenames in walk(start_path):
+                        for f in filenames:
+                            fp = pjoin(dirpath, f)
+                            # skip if it is symbolic link
+                            if not islink(fp):
+                                total_size += getsize(fp)
+
+                    return total_size
+                dataset_sizes_and_modified_times = []
+                for dataset_cache_dir in listdir(self.cache_dir):
+                    dataset_cache_path = pjoin(self.cache_dir, dataset_cache_dir)
+                    dataset_sizes_and_modified_times.append({"path": dataset_cache_path, "size": get_size(dataset_cache_path), "modified_time": stat(dataset_cache_path).st_mtime})
                 repo = Repository(local_dir=self.cache_path, clone_from="datameasurements/" + self.dataset_cache_dir, repo_type="dataset", use_auth_token=HF_TOKEN)
+
+                max_bytes = 8000000000  # 8 gigabytes.
```

Comment:
Where is this value coming from? Is it one users could/should set?
Either way, since it's a constant, can we put this as a constant at the top of the file (e.g., _MAX_BYTES=8000000000)?